src/book/03-guides/04-general/04-documentation.mdx

```diff
@@ -0,0 +1,82 @@
+---
+section: Guides
+chapter: General
+title: Documentation
+description: How to properly document code.
+slug: /guides/general/documentation
+---
+
+## Conventions
+
+We use tokens called "directives" or "tags" to annotate different parts of a comment with rich metadata. These directives start with `@` or `\\` (`@` is preferred) and are followed by the directive name, arguments (for some directives), and a single line of description.
+
+When writing modules, the `.hpp` should include a documenting comment for each class, struct, function, method, global variable, member variable and anything extra that you think may need one.
+
+### Directives
+
+These are some common directives:
+
+| `@brief` | A brief description of the following code. Every documenting comment should have this. If the first line does not have a directive it is treated as `@brief`. |
```

Comment:
This formatting is strange. have you considered a table?

Reply:
This is meant to be a table